openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -833,6 +834,7 @@ def _delete_cookie(request, response):
         secure=settings.SESSION_COOKIE_SECURE or None,
         httponly=settings.SESSION_COOKIE_HTTPONLY or None,
     )
+    delete_logged_in_cookies(response)
```

Comment:
Can we update the docstring of this method to specify exactly which cookies are being deleted? Also it may be worth a unit test to make sure delete_logged_in_cookies gets called. if someone takes out this code we'll probably want to know about it.